Habitica/src/main/java/com/habitrpg/android/habitica/data/implementation/ApiClientImpl.java

```diff
@@ -127,15 +127,15 @@
 
 public class ApiClientImpl implements Action1<Throwable>, ApiClient {
     private final GsonConverterFactory gsonConverter;
-    private final HostConfig hostConfig;
-    private final Retrofit retrofitAdapter;
+    private HostConfig hostConfig;
+    private Retrofit retrofitAdapter;
     private final PopupNotificationsManager popupNotificationsManager;
 
     private CrashlyticsProxy crashlyticsProxy;
     private Context context;
 
     // I think we don't need the ApiClientImpl anymore we could just use ApiService
-    private final ApiService apiService;
+    private ApiService apiService;
 

```

Comment:
Making those non-final is surely not what we want, but I couldn't find another way to do this, "thanks" to dependency injection.